src/AdsImporter/Database/MongoMigration.php

```diff
@@ -114,19 +124,52 @@ public function addBlock(Block $block): void
         }
     }
 
-    public function addTransaction(AbstractTransaction $transaction): void
+    public function addTransaction(ArrayableInterface $transaction): void
     {
-        $document = [
-            "id" => $transaction->getId(),
-            "size" => $transaction->getSize(),
-            "type" => $transaction->getType()
-        ];
+        $document = $transaction->toArray();
 
         try {
             $this->transactionCollection->insert($document);
         } catch (\Exception $ex) {
             // do nothing when a block exists in the database
         }
+
+        if ($transaction instanceof SendOneTransaction) {
+            $data = [];
+
+            $data[] = [
+                "accountId" => $transaction->getSenderAddress(),
+                "transactionId" => $transaction->getId(),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "transactionId" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)